xml_inscriptions/transcribed/MA.North.AC.MAM.L.AC.2012.346.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?><?oxygen RNGSchema="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="xml"?><TEI xmlns:xi="http://www.w3.org/2001/XInclude" xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader><!-- testing1 -->
+      <fileDesc>
+         <titleStmt>
+            <!--printable version of USEpi inscription ID:-->
+            <title>MA.North.AC.MAM.L.AC.2012.346</title>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Brown University</authority>
+            <idno type="USEpigraphy" xml:id="MA.North.AC.MAM.L.AC.2012.346">MA.North.AC.MAM.L.AC.2012.346</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>MA</region>
+                  <settlement>North</settlement>
+                  <institution>AC</institution>
+                  <repository>MAM</repository>
+                  <idno>AC.2012.346</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="lat"/>
+                  <msItem class="#funerary.epitaph">
+                     <p>epitaph of Laberia Alexandria</p>
+                  </msItem>
+               </msContents>
+               <physDesc>
+                  <objectDesc ana="#sarcophagus">
+                     <supportDesc ana="#stone.marble">
+                        <support><p>Carrara marble sarcophagus</p><dimensions type="surface" unit="cm">
```

Comment:
I think as a rule we're trying to minimize details like "Carrara" here; it can go in the commentary, but for here we just want marble.